### ISSUE TITLE
Dividing two times should return a scalar, not a time

### ISSUE THREAD

**Issue by DumpsterDoofus** [SFML/SFML.Net#177]
In `Time.cs`, division of two times is defined like this:

```
public static Time operator /(Time left, Time right) => FromMicroseconds(left.AsMicroseconds() / right.AsMicroseconds());
```

However, dimensional analysis suggests dividing two times should be a scalar, not a time. In physics, 3 minutes/2 minutes is the scalar 1.5, not the time 1.5 microseconds.

I think this makes more sense:

```
public static float operator /(Time left, Time right) => left.AsSeconds() / right.AsSeconds();
```

Every other operator in `Time.cs` seemed sensible, except for this one. What do you think? I can file a PR if you think this makes sense.

**Comment by eXpl0it3r** [comment]
This is indeed incorrect, SFML's Time/Time division [yields a float](https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1Time.php#a097cf1326d2d50e0043ff4e865c1bbac), as such the .Net API should also return a floating point value (probably decimal?).

**Comment by DemoXinMC** [comment]
This would constitute a change to the API, how do we feel this change should be made?

**Comment by eXpl0it3r** [comment]
@LaurentGomila how do you want to handle breaking .Net API changes for stuff that were added by mistake?

**Comment by LaurentGomila** [comment]
This operator is not widely used, and since it incorrectly returns a Time instance, it may not even be used at all, so I don't think we take a huge risk by fixing it right now.

**Comment by eXpl0it3r** [comment]
Fixed with #178
